packages/amplify_datastore/example/lib/models/Blog.dart

```diff
@@ -37,7 +37,16 @@ class Blog extends Model {
   }
 
   String get name {
-    return _name!;
+    try {
```

Comment:
*Question:* Do you know why some of the existing models were changed? I am assuming they were generated with one of the pre-release versions of null-safety codgen and were not updated?

Reply:
Thanks for double checking! I think the original files became inaccurate at all due to merging processes, and yes, the original files seemed generated with in-development codegen.